package/yast2.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Tue Jul 28 10:50:05 UTC 2015 - ancor@suse.com
+
+- Properly display error information when running in command line
```

Comment:
"Error" can mean many things. AFAIK the fix concerns only exceptions, so we should say 
> Do not hide exceptions when running in command line mode (boo#939565).

 right?